intershop/iom-project-archetype
# Task/78057 integrate latest changes

testframework
* added testframework-config.user.yaml to .gitignore
* added dependencies for testframework
* added integration tests to verify goal
partner devops
* add azure-pipelines.yml
* renamed maven repository to iom-maven-artifacts
* renamed intershop docker repo to docker.tools.intershop.com/iom/intershophub/
new IOM version
* removed obsolete properties is.oms.mail.internal.* from cluster.properties
* updated platform version to 4.1.3
CI:
* renamed testproject from blueprint to archetype
* changed usage of ACRs to get in line with latest changes
* removed local software installations
* use Helm chart from Chart repo (was git repo before)
* changed usage of Maven repos to get in line with Partner DevOps Environment
Doc:
* added integration into Partner DevOps Environment
